app/models/scorebook.rb

```diff
@@ -0,0 +1,97 @@
+class Scorebook
```

Comment:
@kriskelly liking the OOP town. Perhaps we can put this in a folder app/queries ?